contrib/storage-phoenix/pom.xml

```diff
@@ -313,98 +200,28 @@
           <groupId>log4j</groupId>
           <artifactId>log4j</artifactId>
         </exclusion>
-        <exclusion>
-          <groupId>org.slf4j</groupId>
-          <artifactId>*</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>javax.servlet</groupId>
-          <artifactId>servlet-api</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>io.netty</groupId>
-          <artifactId>netty</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>com.zaxxer</groupId>
-          <artifactId>HikariCP-java7</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>org.apache.commons</groupId>
-          <artifactId>commons-csv</artifactId>
-        </exclusion>
       </exclusions>
     </dependency>
+
+
   </dependencies>
+
   <build>
     <plugins>
-      <plugin>
-        <artifactId>maven-resources-plugin</artifactId>
-        <executions>
-          <execution>
-            <id>copy-java-sources</id>
-            <phase>process-sources</phase>
-            <goals>
-              <goal>copy-resources</goal>
-            </goals>
-            <configuration>
-              <outputDirectory>${basedir}/target/classes/org/apache/drill/exec/store/phoenix</outputDirectory>
-              <resources>
-                <resource>
-                  <directory>src/main/java/org/apache/drill/exec/store/phoenix</directory>
-                  <filtering>true</filtering>
-                </resource>
-              </resources>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
           <skipTests>${skipTests}</skipTests>
-          <forkCount combine.self="override">1</forkCount>
+          <forkCount>1</forkCount>
           <reuseForks>false</reuseForks>
-          <includes>
-            <include>**/PhoenixTestSuite.class</include>
-            <include>**/SecuredPhoenixTestSuite.class</include>
-          </includes>
-          <excludes>
-            <exclude>**/*Test.java</exclude>
-          </excludes>
-          <argLine>-Xms2048m -Xmx2048m</argLine>
+          <argLine>
+            -Xms2048m -Xmx2048m
+            --add-opens=java.base/java.lang=ALL-UNNAMED
+            --add-opens=java.base/java.util=ALL-UNNAMED
+          </argLine>
         </configuration>
       </plugin>
     </plugins>
   </build>
-  <profiles>
-    <profile>
```

Comment:
Are we going to drop the Hadoop 2 profile because it's become a security liability (I'm in favour). I.e. we're saying that 1.22.0 was shipped with no Hadoop 2 builds but you still have the ability to roll your own from source, but in anything after 1.22.0 we won't even offer source code support for Hadoop 2?

Reply:
Hadoop 2 users can simply use older Drill releases. If the care so little about the old Hadoop version and its multiple insecure libs then why should they care much about latest fixes in Drill?

Reply:
I agree and there's also the reality that the project simply does not have the resources to simultaneously
1. keep Hadoop 2 in the code base and
2. keep Drill patched against vulnerabilities.

Reply:
OK.. Let's do this.  Can we merge this, and we'll create a new PR to remove Hadoop 2 support.